concent_api/core/tests/test_unit_validation.py

```diff
@@ -112,8 +113,8 @@ def test_that_function_should_pass_when_value_is_alphanumeric(self):
     def test_that_function_should_raise_exception_when_value_is_not_alphanumeric(self):
```

Comment:
BTW: Could you please make this class inherit from `unittest.TestCase`? There is no need for heavy `ConcentIntegrationTestCase` base.

Reply:
Done